app/index.dev.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
+import { Provider } from 'react-redux';
+import Redbox from 'redbox-react';
+import configureStore from './configure_store';
+import Root from './root';
+
+const render = (Component) => {
+  ReactDOM.render(
+    <AppContainer errorReporter={ Redbox }>
+      <Provider store={ configureStore() }>
+        <Component/>
+      </Provider>
+    </AppContainer>,
+    document.querySelector('#root'),
+  );
+};
+
+render(Root);
+
+if (module.hot) {
+  module.hot.accept('./root', () => {
+    const NewRoot = require('./root').default;
+    render(NewRoot);
+  });
+}
```

Comment:
This is related to hot module reloader. It will reload components from disk when needed (when they will be saved in editor). So no need to press Cmd+R, it will refresh only changed components if it's possible, or the whole page if not.